api/src/main/java/com/vimeo/networking2/requests/ApiResponseCallback.kt

```diff
@@ -0,0 +1,14 @@
+package com.vimeo.networking2.requests
+
+import com.vimeo.networking2.ApiError
+
+/**
+ * Informs you of the result of an API request. Provides the response or any errors.
+ */
+interface ApiResponseCallback<ResponseType_T> {
```

Comment:
Should this be using the enumerated `ApiResponse` instead of having singular success and error methods?